doc/source/structures/celestial_bodies/body.rst

```diff
@@ -9,11 +9,6 @@ This is any sort of planet or moon. To get a variable referring to a Body, you c
     // like "Mun" for example.
     SET MY_VAR TO BODY("name").
 
-.. note::
-    .. versionchanged:: 0.13
-        A Celestial Body is now also an :ref:`Orbitable <orbitable>`, and can use all the terms described for these objects too.
-
-
```

Comment:
Disagree-ish:  Go ahead and remove the mention of version number and the phrasing "is now", sure, but don't delete the fact that the documentation is telling the reader that Celestial Bodies are a kind of Orbitable.

Reply:
Added that paragraph in as a sentence.  Incidentally, I don't like the duplication of the explanations in that section but I do not currently see a great way of addressing it.